contractcourt/chain_watcher.go

```diff
@@ -749,6 +781,15 @@ func (c *chainWatcher) dispatchContractBreach(spendEvent *chainntnfs.SpendDetail
 		LocalChanConfig:         c.cfg.chanState.LocalChanCfg,
 	}
 
+	// Attempt to add a channel sync message to the close summary.
+	chanSync, err := c.chanSyncMsg()
+	if err != nil {
+		log.Errorf("ChannelPoint(%v): unable to create channel sync "+
+			"message: %v", c.cfg.chanState.FundingOutpoint, err)
+	} else {
+		closeSummary.LastChanSyncMsg = chanSync
+	}
+
```

Comment:
Unrelated to this PR, but shouldn't the `BRAR` be the one that's making this DB state here? Re the whole reliable hand off thing.

Reply:
the brar writes the retributions to disk, then acks back. after that the chain watcher knows it's safe to mark the channel as pending closed

Reply:
We could let the `BRAR` have that responsibility, as we could get rid of ACK logic.

Reply:
the cleaner solution IMO is just to write resolutions in `CloseChannel`. then there is no need for a reliable handoff. the resolutions can be passed in memory to kick off the process, but if we crash the brar would look for all breached channels in the db and load resolutions from there